omnibus/files/private-chef-ctl-commands/global_admins.rb

```diff
@@ -0,0 +1,139 @@
+#
+# Copyright 2015 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'restclient'
+require 'json'
+require 'sequel'
+
+PLACEHOLDER_GLOBAL_ORG_ID = "00000000000000000000000000000000"
+
+add_command_under_category "grant-global-admin-permissions", "global-admins", "Grant a user the ability to create other users by added the user to the global-admins group.", 2 do
+
+  cmd_args = ARGV[3..-1]
+  if cmd_args.length != 1
+    msg = "Username is the only argument to grant-global-admin-permissions.\nPlease pass a single argument."
+    STDERR.puts msg
+    raise SystemExit.new(1, msg)
```

Comment:
return exit! 1.  exit! in this case is a Ctl method This also makes sure that post-hooks run or don't run based on the return. Same for the other usage of SystemExit.